src/bson.c

```diff
@@ -35,6 +35,30 @@
 #define PHONGO_IS_CLASS_INSTANTIATABLE(ce) \
 	(!(ce->ce_flags & (ZEND_ACC_INTERFACE|ZEND_ACC_IMPLICIT_ABSTRACT_CLASS|ZEND_ACC_EXPLICIT_ABSTRACT_CLASS)))
 
+#if PHP_VERSION_ID >= 70000
+# define PHONGO_BSON_STATE_ZCHILD(state) (&((php_phongo_bson_state *)(state))->zchild)
+#else
+# define PHONGO_BSON_STATE_ZCHILD(state) (((php_phongo_bson_state *)(state))->zchild)
+#endif
+
+#if PHP_VERSION_ID >= 70000
+# define PHONGO_BSON_TYPE_WRAPPER(wrapper_ce, zchild) \
+	if (wrapper_ce) { \
+		zval zwrapper; \
+		zend_call_method_with_1_params(NULL, wrapper_ce, NULL, "createfrombsontype", &zwrapper, &(zchild)); \
```

Comment:
Confirmed that `zend_call_method()` allows `function_name` to have any case when invoking a method on an object (specifying the first `obj` arg); however, invoking a static method when only supplying `obj_ce` requires the function name to be lowercase. I assume this has something to do with differences in storing the function entries on the CE and object instances.

In any event, these strings are only used twice in these two macros, unlike the `bsonSerialize` and `bsonUnserialize` constants, so I will not create a new constant.